src/Sharpbrake.Extensions.Logging/AirbrakeLogger.cs

```diff
@@ -0,0 +1,108 @@
+    /// </summary>
+    public class AirbrakeLogger : ILogger
+    {
+        private readonly AirbrakeNotifier notifier;
+        private readonly IHttpContextAccessor contextAccessor;
+        private readonly LogLevel minLevel;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AirbrakeLogger"/> class.
+        /// </summary>
+        public AirbrakeLogger(AirbrakeNotifier notifier, IHttpContextAccessor contextAccessor, LogLevel minLevel)
+        {
+            this.notifier = notifier ?? throw new ArgumentNullException(nameof(notifier));
+            this.contextAccessor = contextAccessor;
+            this.minLevel = minLevel;
+        }
+
+        /// <summary>
+        /// Notifies the Airbrake endpoint on an exception.
+        /// </summary>
+        public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+        {
+            if (exception == null)
+                return;
+
+            notifier.NotifyAsync(exception, GetHttpContext(), GetErrorSeverity(logLevel));
+        }
+
+        /// <summary>
+        /// Checks if the given <paramref name="logLevel" /> is enabled.
+        /// </summary>
+        public bool IsEnabled(LogLevel logLevel)
+        {
+            return logLevel >= minLevel;
+        }
+
+        /// <summary>
+        /// Begins a logical operation scope.
+        /// </summary>
+        /// <param name="state">The identifier for the scope.</param>
+        /// <returns>An IDisposable that ends the logical operation scope on dispose.</returns>
+        public IDisposable BeginScope<TState>(TState state)
+        {
+            // TODO: Consider what scopes may look like in our case... One option: a bunch of exceptions
+            // that are sent to the dashboard in a single call.
+            return NoopDisposable.Instance;
+        }
+
+        /// <summary>
+        /// Gets the current HTTP context.
+        /// </summary>
+        private IHttpContext GetHttpContext()
+        {
+            var httpContext = contextAccessor?.HttpContext;
+            return httpContext == null ? null : new AspNetCoreHttpContext(httpContext);
+        }
+
+        /// <summary>
+        /// Maps <see cref="LogLevel"/> onto <see cref="Severity"/> of the error.
+        /// </summary>
+        private static Severity GetErrorSeverity(LogLevel logLevel)
+        {
+            if (logLevel == LogLevel.Critical)
+                return Severity.Critical;
+
+            if (logLevel == LogLevel.Error)
+                return Severity.Error;
+
+            if (logLevel == LogLevel.Warning)
+                return Severity.Warning;
+
+            if (logLevel == LogLevel.Information)
+                return Severity.Info;
+
+            if (logLevel == LogLevel.Debug)
+                return Severity.Debug;
+
+            if (logLevel == LogLevel.Trace)
+                return Severity.Notice;
```

Comment:
Would it make sense to specify a table of log levels from Logging that correspond to Airbrake log levels? We support more than 6 log levels, so users with that knowledge might not know that Trace defaults to Notice.

Reply:
It makes sense. Perhaps it should be done for other integrations as well. Could we postpone that for another PR? I would like to perform some additional changes to documentation as well.

Reply:
Sounds good to me.